assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$button-color: #f68e1f;
```

Comment:
I realise this needs more discussion that just a Github comment -- but is it really too limiting to use web safe colours in our design?  More a question for @jetpacksandrollerskates